src/ray/gcs/format/gcs.fbs

```diff
@@ -49,6 +52,11 @@ table ObjectTableData {
   is_eviction: bool;
   // The number of times this object has been evicted from this node so far.
   num_evictions: int;
+  // Whether this object was created by ray.put.
+  is_put: bool;
```

Comment:
What do we need this for?

Reply:
Based on discussions with @concretevitamin's, this mitigates the need to deduplicate object ids in `monitor.py`.

Reply:
Ok, I see how it's being used now, but I don't think it is worth adding the additional information to the GCS. Since we're already pulling down all of the task table, we can just look up the relevant task and check the driver to see if the object corresponds to the given job.

Reply:
I agree that we can remove this from the table. I'm going to deduplicate the object ids instead of checking for `is_put`.